src/tests/ftest/aggregation/dfuse_space_check.py

```diff
@@ -89,52 +96,50 @@ def test_dfusespacecheck(self):
             return all the space back.
             Now create small files again until pool is out of space and check,
             whether out of space happens at the same file count as before.
-        :avocado: tags=all,hw,daosio,small,full_regression,dfusespacecheck
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,small
+        :avocado: tags=aggregation,daosio,dfuse
+        :avocado: tags=dfusespacecheck
         """
         # get test params for cont and pool count
-        self.block_size = self.params.get("block_size",
-                                          '/run/dfusespacecheck/*')
+        self.block_size = self.params.get('block_size', '/run/dfusespacecheck/*')
 
-        # Create a pool, container and start dfuse.
+        # Create a pool, container, and start dfuse
         self.create_pool()
         self.create_cont()
         self.start_dfuse(self.hostlist_clients, self.pool, self.container)
 
-        # get scm space before write
-        self.space_before = self.get_nvme_free_space()
+        # get nvme space before write
+        self.initial_space = self.get_nvme_free_space()
 
-        # create large file and perform write to it so that if goes out of
-        # space.
-        large_file = str(self.dfuse.mount_dir.value + "/" + "largefile.txt")
-        cmd = "touch {}".format(large_file)
-        self.execute_cmd(cmd)
-        dd_count = ((self.space_before / self.block_size) + 1)
+        # Create a file as large as we can
+        large_file = os.path.join(self.dfuse.mount_dir.value, 'largefile.txt')
+        self.execute_cmd('touch {}'.format(large_file))
+        dd_count = (self.initial_space // self.block_size) + 1
         write_dd_cmd = "dd if=/dev/zero of={} bs={} count={}".format(
             large_file, self.block_size, dd_count)
         self.execute_cmd(write_dd_cmd, False)
 
-        # store free space after write and remove the file
-        rm_large_file = "rm -rf {}".format(large_file)
-        self.execute_cmd(rm_large_file)
+        # Remove the file
+        self.execute_cmd('rm -rf {}'.format(large_file))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string